extmod/uasyncio/tcp.py

```diff
@@ -0,0 +1,76 @@
+# MicroPython uasyncio module
+# MIT license; Copyright (c) 2019-2020 Damien P. George
+
+from . import core
+
+
+# Create a TCP stream connection to a remove host
+async def open_connection(host, port):
+    import usocket as socket
+
+    try:
+        from uerrno import EINPROGRESS
+    except:
+        EINPROGRESS = 115  # Use standard built-in value if uerrno doesn't provide it
```

Comment:
Wouldn't it be better to error out here and go fix the ports that don't define EINPROGRESS? See #5752

Reply:
I agree.  It's rather dangerous to keep this an integer because if it's wrong then it won't behave correctly and won't error out.  Now changed.

The only port I know of that doesn't define this is cc3200 which can be fixed.